src/form/ColorMapInput.tsx

```diff
@@ -115,6 +117,11 @@ const ColorMapInput: React.FC<ColorMapProps & InjectedIntlProps> = (props) => {
   });
   const [showCustomColormapModal, setShowCustomColormapModal] = useState(false);
```

Comment:
Since you kept the `showColomapModal` in the raster layer form, it's unnecessary to also create a new one here in the ColormapInput component. Then you don't have to keep them in sync with each other in the below function.